app/perf_overlay.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_PERFORMANCE_COUNTERS=y
```

Comment:
@btian1 Can you split this patch into 2 separate patches with the first one to introduce the macros and the second one for enabling perf build?

Reply:
Also, Can you please fix your name in the sign-off?

Reply:
let me check, I already tried to fix sign off.
ok, seems change too much in one patch is not a good way , let me split based on you, marc, Liam's requirement

Reply:
@andrula-song this overlay is not used by default.

Reply:
@btian1 The signed-off is still wrong, you should have proper full name there. 
Also, I recommend @ranj063 's idea of splitting to two patches. One to move the task perf tracking to new interface and a separate patch to add the performance build.